lsix

```diff
@@ -203,7 +203,7 @@ main() {
 	for arg; do
 	    if [ -d "$arg" ]; then
 		echo Recursing on $arg
-		(cd "$arg"; $lsix)
+		(cd "$arg" || exit; $lsix)
```

Comment:
Exiting abruptly like that seems wrong. Maybe continue, instead.

Reply:
You're right in this case because it's already checked that there will be a directory to be able to "cd" in…